dali/base/dasess.cpp

```diff
@@ -41,32 +45,62 @@ using namespace cryptohelper;
 #pragma warning (disable : 4355)
 #endif
 
+static std::unordered_map<std::string, DaliClientRole> daliClientRoleMap = {
```

Comment:
Is there not such a thing as a bidirectional map?

Reply:
Not a STL one, I think there might be a boost impl.